tests/functional/ionIonBeam/ion_ion_beam1d.py

```diff
@@ -52,10 +55,10 @@ def bx(x):
         return 1.
 
     def by(x):
-        return 0.
+        return 0
```

Comment:
not sure why the return should be `0` and not `0.`

Reply:
There's no particular reason indeed, both work so we could keep `0.`